src/utils/getArtworks.ts

```diff
@@ -0,0 +1,89 @@
+import "dotenv/config"
+// import { Connection, createConnection, getMongoRepository } from "typeorm"
+
+// import { TableCheck } from "typeorm/schema-builder/table/TableCheck"
+// import { databaseConfig } from "../config/database"
+import { Collection } from "../Entities"
+import { search } from "../lib/search"
+
+// const { NODE_ENV } = process.env
+
+/**
+ * Gets artworks associated with a collection
+ */
+
+export const getArtworks = async (collection: Collection) => {
+  const { query } = collection
+  const { artist_ids, gene_ids, keyword, tag_id } = query
+
+  const subQuery: any[] = []
+
+  if (keyword) {
+    subQuery.push({
+      multi_match: {
+        type: "most_fields",
+        fields: "name.* genes.*^4 tags.*^4 auto_tags.*^2 partner_name.*^2 artist_name.*^2".split(
+          " "
+        ),
+        query: keyword,
+      },
+    })
+
+    subQuery.push({
+      bool: {
+        should: {
+          match_phrase: {
+            name: keyword,
+          },
+        },
+      },
+    })
+  }
+
+  if (artist_ids && artist_ids.length) {
+    subQuery.push({
+      terms: {
+        artist_id: artist_ids,
+      },
+    })
+  }
+
+  if (gene_ids && gene_ids.length) {
+    subQuery.push({
+      term: {
+        gene_ids,
+      },
+    })
+  }
+
+  if (tag_id) {
+    subQuery.push({
+      term: {
+        tags: tag_id,
+      },
+    })
+  }
+
+  const response = await search.client.search({
+    index: `artworks_staging`,
```

Comment:
The name of the artworks index is actually `gravity_application_staging/production` (that's a legacy thing from when we only had artworks in ES and so didn't have a better index name specified)

Reply:
thanks for the note @mzikherman !

Reply:
@mzikherman I checked our ES cluster config & it says `artworks_staging` is an alias for `gravity_application_staging` so they point to the same index? and the query works

Reply:
Ah, I didn't know we had that setup? In that case 👌